modules/backend/constants/queryOptions.ts

```diff
@@ -178,5 +178,4 @@ export const vehiclesOptions = () =>
   queryOptions({
     queryKey: ['Vehicles'],
     queryFn: () => clientApi.vehiclesControllerVehiclesGetMany(),
-    select: (res) => res.data,
   })
```

Comment:
I would maybe add a comment why you don't use `select: (res) => res.data`, otherwise, I think people will try to implement it :D